Product/SoapUI_Test/RegressionSuite/Bimodal/AdapterPatientDiscoveryAsyncReq/AdapterPatientDiscoveryAsyncReq-soapui-project.xml

```diff
@@ -847,8 +847,8 @@ com.eviware.soapui.impl.wsdl.testcase.WsdlTestRunContext.metaClass.setGatewaySta
   def env = [
     (JMXConnector.CREDENTIALS): (String[])[username, password]
   ];
-  def serverUrl = 'service:jmx:rmi://' + jmxHost + ':' + jmxPort + '/jndi/rmi://' + jmxHost + ':' + jmxPort + 
-'/jmxrmi';
+  //def serverUrl = 'service:jmx:rmi://' + jmxHost + ':' + jmxPort + '/jndi/rmi://' + jmxHost + ':' + jmxPort + '/jmxrmi';
```

Comment:
Can we remove the commented line here. Also does jmx need not use jndi name now?